src/libraries/System.Runtime.Serialization.Schema/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
+    <UnsupportedOSPlatforms>browser;ios;tvos;maccatalyst</UnsupportedOSPlatforms>
+  </PropertyGroup>
```

Comment:
How did you come up with this list? If ios isn't supported, why would android be?

Reply:
These are the <UnsupportedOSPlatforms> from System.CodeDom. The build system isn't smart enough or not willing to assume this list on it's own.